domain/schema/controller.go

```diff
@@ -213,6 +223,52 @@ CREATE TABLE cloud (
         REFERENCES        cloud_type(id)
 );
 
+-- v_cloud is used to fetch well construct information about a cloud. This view
+-- also includes information on if the cloud is the controller models cloud.
```

Comment:
... on whether the cloud is the controller model's cloud.

Reply:
Done & Done